docs/sources/http_api/access_control.md

```diff
@@ -349,14 +354,15 @@ Content-Type: application/json
 
 #### JSON body schema
 
-| Field Name  | Data Type           | Required | Description                                                         |
-| ----------- | ------------------- | -------- | ------------------------------------------------------------------- |
-| version     | number              | Yes      | Version of the role. Must be incremented for update to work.        |
-| name        | string              | Yes      | Name of the role.                                                   |
-| description | string              | No       | Description of the role.                                            |
-| displayName | string              | No       | Display name of the role, visible in the UI.                        |
-| group       | string              | No       | The group name the role belongs to.                                 |
-| permissions | List of Permissions | No       | The full list of permissions the role should have after the update. |
+| Field Name  | Data Type           | Required | Description                                                                                                                                                                    |
+| ----------- | ------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| version     | number              | Yes      | Version of the role. Must be incremented for update to work.                                                                                                                   |
+| name        | string              | Yes      | Name of the role.                                                                                                                                                              |
+| description | string              | No       | Description of the role.                                                                                                                                                       |
+| displayName | string              | No       | Display name of the role, visible in the UI.                                                                                                                                   |
+| group       | string              | No       | The group name the role belongs to.                                                                                                                                            |
+| hidden      | boolean             | No       | A flag indicating if the role is hidden or not. If set to `true`, role will not appear in the role picker and will not be listed by API endpoints unless explicitly specified. |
```

Comment:
```suggestion
| hidden      | boolean             | No       | Specify whether the role is hidden or not. If set to `true`, then the role does not show in the role picker. It will not be listed by API endpoints unless explicitly specified. |
```